src/engine/Engine.ts

```diff
@@ -788,10 +788,14 @@ O|===|* >________________>\n\
        */
       public setAntialiasing(isSmooth: boolean) {
          this._isSmoothingEnabled = isSmooth;
-         (<any>this.ctx).imageSmoothingEnabled = isSmooth;
-         (<any>this.ctx).webkitImageSmoothingEnabled = isSmooth;
-         (<any>this.ctx).mozImageSmoothingEnabled = isSmooth;
-         (<any>this.ctx).msImageSmoothingEnabled = isSmooth;
+
+         var ctx: any = this.ctx;
+         ctx.imageSmoothingEnabled = isSmooth;
+         ['webkitImageSmoothingEnabled', 'mozImageSmoothingEnabled', 'msImageSmoothingEnabled'].forEach((smoothing) => {
```

Comment:
It's tiny, but using `forEach` does incur anonymous function creation overhead, we could do:

```
for (var smoothing of [arr]) {
}
```

instead for speed.